### ISSUE TITLE
Create map in Topolograph automatically

### ISSUE THREAD

**Issue by consultoriaheitor** [Vadims06/ospfwatcher#8]
Hello everyone, I recently installed ospfwatcher+elastic+topolograph, but I noticed that with the change in OSPF topology, I did not see it creating a new map in Topolograph. Should it be creating a new map when the topology changes, or is there a specific situation in which it would recreate the topology in Topolograph?

**Comment by Vadims06** [comment]
Hi @consultoriaheitor ,
thank you for using OSPF Watcher.
OSPF Watcher should save a topology right after you start it via `docker-compose up -d`. I advise you to check containers status via `docker ps`. What is the status of `ospfwatcher`? Please run `docker logs ospfwatcher`.

I've noticed that topolograph [compose file](https://github.com/Vadims06/topolograph-docker/blob/master/docker-compose.yml) was not updated. Please update it via
```
cd /topolograph-docker
docker-compose down
git pull
docker-compose up -d
````
Also please check [these steps](https://github.com/Vadims06/ospfwatcher?tab=readme-ov-file#troubleshooting) and let me know what steps you passed.

**Comment by consultoriaheitor** [comment]
Hello, it is working and has created a map in Topolograph. However, there have been changes in the network, and the existing map is now outdated. Is there a feature that allows for automatic regeneration of the map when there are changes in the network topology, or if the topology differs for a certain period, say X time? Alternatively, is there a way to update the already created map to reflect these changes?

**Comment by consultoriaheitor** [comment]
Yes, I would like OSPF network changes, such as adding a new host, to automatically create a new map. If the topology remains altered for more than 24 hours (just a suggested time frame, as temporary disruptions could lead to non-permanent changes), the existing topology file should update. This could reduce the number of different files for minor changes. Another idea is to add an 'Update Now' button in Topolograph, or select triggers for overwriting the current map, which would be very useful. Additionally, it would be great to name the maps and add comments on changes, like 'Traffic engineering applied to reduce bandwidth on link X' or 'Added POP Z'.

**Comment by Vadims06** [comment]
@consultoriaheitor,
I added new nodes, edges on a topology map under Monitoring tab and it's available in https://github.com/Vadims06/topolograph/releases/tag/v2.38.
I also created a feature request to update a topology on the main page from watcher's data
